Templates/FinniversKitView.xctemplate/___FILEBASENAME___.swift

```diff
@@ -18,14 +18,21 @@ class ___FILEBASENAMEASIDENTIFIER___: <#View Class#> {
         setup()
     }
 
-    private func setup() {
+    // MARK: - Superclass Overrides
+    
+}
+
+// MARK: - Private
+
+private extension ___FILEBASENAMEASIDENTIFIER___ {
+    func setup() {
         // Perform setup
         // Add child views as subviews
         // Setup constraints/frames
     }
+}
 
-    // MARK: - Superclass Overrides
-
-    // MARK: - Private
+// MARK: - Public
 
+public extension ___FILEBASENAMEASIDENTIFIER___ {
```

Comment:
Can we have the public extension come before the private? Makes it easier when you open a view and start looking for what it can do.

Reply:
I agree, thats better 👍